.github/workflows/container-build-test-publish.yml

```
@@ -50,10 +51,13 @@ jobs:
           if [[ "${{ steps.filter.outputs.shared }}" == "true" || "${{ steps.filter.outputs.root-files }}" == "true" || "${{ steps.filter.outputs.workflows }}" == "true" || "${{ steps.filter.outputs.docker }}" == "true" || "${{ steps.filter.outputs.security }}" == "true" ]]; then
             echo "🔄 Shared files or critical files changed - processing all containers"
             echo "Changed files: shared=${{ steps.filter.outputs.shared }}, root-files=${{ steps.filter.outputs.root-files }}, workflows=${{ steps.filter.outputs.workflows }}, docker=${{ steps.filter.outputs.docker }}, security=${{ steps.filter.outputs.security }}"
-            containers=("bunkbot" "djcova" "starbunk-dnd" "covabot")
+            containers=("bunkbot" "djcova" "starbunk-dnd" "snowbunk" "covabot")
             echo "test-shared=true" >> $GITHUB_OUTPUT
 
             # Log specific shared changes for better debugging
+
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 58: did not find expected key

(syntax-check)